src/engine/Input/Pointer.ts

```diff
@@ -80,6 +86,44 @@ export class PointerEvent extends GameEvent<any> {
 };
 
 /**
+ * Wheel Events
+ *
+ * Represents a mouse wheel event. See [[Pointers]] for more information on
+ * handling point input.
+ */
+export class WheelEvent extends GameEvent<any> {
```

Comment:
Do you think it'd be helpful to provide a `direction` property for developers to quickly determine direction? Or is comparing positive to negative enough?